packages/rum/src/index.d.ts

```diff
@@ -108,6 +108,8 @@ declare module '@elastic/apm-rum' {
       payload?: string
       headers?: Record<string, string>
     }) => boolean
+    transactionContextCallback?: (...args: any[]) => any
+    spanContextCallback?: (...args: any[]) => any
```

Comment:
let's be more explicit about the types here:
- we know for sure the RUM agent will call them with no args
- we have an explicit type for labels

```suggestion
    transactionContextCallback?: () => Labels
    spanContextCallback?: () => Labels
```